ci-operator/step-registry/gcp/provision/bastionhost/gcp-provision-bastionhost-commands.sh

```diff
@@ -74,7 +74,8 @@ MACHINE_TYPE="n2-standard-2"
 echo "${REGION}" >> "${SHARED_DIR}/proxyregion"
 
 bastion_name="${CLUSTER_NAME}-bastion"
-gcloud compute instances create "${bastion_name}" \
+CMD="gcloud compute instances create ${bastion_name} \
+  --hostname=${bastion_name}.test.com \
```

Comment:
@jianli-wei Does localhost bring some trouble? Because the bastion would also run proxy container, and the proxy container would share the host's network setting, e.g: /etc/hosts (not sure if the hostname full domain would be written into /etc/hosts), so I am not sure if a full-domain hostname for container host bring some side effect for dns resolving stuff when the bastion is taking a role of proxy during testing. Personally I prefer to keep localhost to be safe.

Reply:
Let me run a test of the proxy scenario.

Reply:
Actually I do not know which exact case would be affected, because you are setting `test.com` as domain, as far as I know the installation part does not use this domain for any testing.

BTW, can you show me /etc/hosts in the bastion host when `--hostname=${bastion_name}.test.com`.

Reply:
FYI the test [rehearse-46111-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.15-installer-rehearse-debug](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/46111/rehearse-46111-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.15-installer-rehearse-debug/1760515791695908864) which uses `cucushift-installer-rehearse-gcp-ipi-proxy-{provision,deprovision}` chains succeeds.

Reply:
@jianli-wei can you show me the content of /etc/hosts in the bastion host when `--hostname=${bastion_name}.test.com`.

Reply:
@jianlinliu FYI
```
[core@ci-op-6nrsns18-5fdca-bastion ~]$ hostname
ci-op-6nrsns18-5fdca-bastion.test.com
[core@ci-op-6nrsns18-5fdca-bastion ~]$ cat /etc/hosts
127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
[core@ci-op-6nrsns18-5fdca-bastion ~]$